src/rgw/rgw_data_sync.h

```diff
@@ -370,6 +372,14 @@ class LatencyConcurrencyControl : public LatencyMonitor {
     auto threshold = (cct->_conf->rgw_sync_lease_period * 1s) / 12;
 
     if (avg_latency() >= 2 * threshold) [[unlikely]] {
+      auto now = ceph::coarse_mono_clock::now();
+      if (now - last_warning > 5min) {
+        ldout(cct, -1)
+            << "WARNING: The OSD cluster is overloaded and struggling to "
+            << "complete ops. You need more capacity to serve this level "
```

Comment:
@cbodley Does this seem like a reasonable warning?

Reply:
yeah, the 5min timer is a nice touch